Documentation/gettingstarted/k8s-install-download-release.rst

```diff
@@ -25,7 +25,7 @@
 
    Download the Cilium release tarball and change to the kubernetes install directory:
 
-    .. parsed-literal::
+   .. parsed-literal::
```

Comment:
didn't include the description in the commit message, but this just fixes the indentation of the section below.